CSS/main.css

```diff
@@ -1,2 +1,72 @@
-.footer{color:pink}
+body {
+  font-family: 'Raleway', sans-serif; }
+
+.section--headline {
+  text-align: center;
+  margin-top: 40px;
+  margin-bottom: 20px; }
+  .section--headline__title {
+    font-size: 34px;
+    text-transform: capitalize; }
+  .section--headline__subtitle {
+    font-size: 18px;
+    color: #4c545f; }
+    .section--headline__subtitle:before {
+      width: 75px;
```

Comment:
znowu powielamy width: 75px; w kilku miejscach - dodajmy to do zmiennej np. "$headline-width-default" czy coś takiego...